drivers/filesystems/fastfat/string.c

```diff
@@ -24,3 +25,10 @@ vfatIsLongIllegal(
 {
     return wcschr(long_illegals, c) ? TRUE : FALSE;
 }
+
+BOOLEAN
+IsDotOrDotDot(PUNICODE_STRING File)
```

Comment:
This could be a `const UNICODE_STRING *` or `PCUNICODE_STRING` since the string isn't modified.

Reply:
Sorry, I do not know what the differences might be. Please tell me your preference or if it can be left as is. These C/C++ types of questions are still magic to me.

Reply:
@ThFabba, Thanks for the approval. My testing shows that this will eliminate over 1500 lines showing the "disk repair" warning and shave about a minute off of the total regression set test time.

Reply:
The addition of `const` will make it a compiler error for this function to try to modify the data pointed to by `File`. Pointers-to-const are generally preferred, unless the data needs to be modified.
So I would recommend the `PCUNICODE_STRING` here (which is a typedef for `const UNICODE_STRING *`).

Reply:
OK. I have made the changes and this does not appear to have changed the calling convention. Thanks.